pipeline/steps/az-shared-config.yml

```diff
@@ -171,6 +171,32 @@ steps:
 
         echo "##vso[task.setvariable variable=pubKey]$pubKey"
 
+  - task: AzureCLI@2
+    displayName: 'Create KV Secrets'
+    inputs:
+      azureSubscription: '${{ parameters.subscriptionName }}'
+      scriptType: 'pscore'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+        $project="Shared Services"
+        $organisation="https://dev.azure.com/hmcts"
+        $env="${{ parameters.env }}"
+        $groupName="cvp-$env-kv"
+        $kvName="$(kvName)"
+
+        Write-Host "##[warning]!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+        Write-Host "##[warning]THIS SCRIPT WILL ONLY WORK WITH NON-SECRET VARIABLES!"
+        Write-Host "##[warning]!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+        
+        echo $(System.AccessToken) | az devops login --organization $organisation
+        $groupId=$(az pipelines variable-group list --org $organisation --project "$project" --query "[?name=='$groupName'].{id:id}[0]" -o tsv --only-show-errors)
+
+        $secrets=$(az pipelines variable-group variable list --org $organisation --project $project --id $groupId) | ConvertFrom-Json -AsHashTable
```

Comment:
The Dynatrace token is a secret value, so should not be plain text in the ADO or passed here plainly